main.go

```diff
@@ -221,6 +463,68 @@ func (e *periodicExporter) setMetrics(ch chan prometheus.Metric) {
 	e.Unlock()
 }
 
+func (e *periodicExporter) scrapeMaster() {
+	stateURL := fmt.Sprintf("%s://%s/master/state.json", e.queryURL.Scheme, e.queryURL.Host)
+
+	log.Debugf("Scraping master at %s", stateURL)
+
+	var state masterState
+
+	err := getJson(&state, stateURL)
+	if err != nil {
+		log.Warn(err)
+		return
+	}
+
+	metrics := make([]prometheus.Metric, 0)
```

Comment:
Same here, just: `metrics := []prometheus.Metric{}` unless you're going to preallocate a slice with a non-zero length (which you could do here theoretically, `len(state.Frameworks) * 3`, but maybe not worth it).